src/core.py

```diff
@@ -92,619 +92,43 @@ def logout(self):
-            result = self.api.get(url_pattern.format(id=hash))
-            return object_type.create(self.api, result)
-        except ObjectNotFoundError:
-            if not raise_not_found:
-                return None
-            else:
-                raise
-
-    def query(self, hash, raise_not_found=True):
-        """
-        Queries for object using provided hash.
-        If you already know type of object you are looking for, use specialized variants:
-
-        - :py:meth:`query_file`
-        - :py:meth:`query_config`
-        - :py:meth:`query_blob`
-
-        .. versionadded:: 2.4.0
-           Added raise_not_found optional argument
-
-        .. versionchanged:: 3.0.0
-           Fallback to :py:meth:`query_file` if other hash than SHA256 was provided
-
-        :param hash: Object hash (identifier, MD5, SHA-1, SHA-2)
-        :type hash: str
-        :param raise_not_found: If True (default), method raises HTTPError when object is not found
-        :type raise_not_found: bool, optional
-        :rtype: :class:`MWDBObject` or None (if raise_not_found=False)
-        :raises: requests.exceptions.HTTPError
-        """
-        if len(hash) != 64:
-            # If different hash than SHA256 was provided
-            return self.query_file(hash, raise_not_found=raise_not_found)
-        return self._query(MWDBObject, hash, raise_not_found)
-
-    def query_file(self, hash, raise_not_found=True):
-        """
-        Queries for file using provided hash
-
-        :param hash: Object hash (identifier, MD5, SHA-1, SHA-2)
-        :type hash: str
-        :param raise_not_found: If True (default), method raises HTTPError when object is not found
-        :type raise_not_found: bool
-        :rtype: :class:`MWDBFile` or None (if raise_not_found=False)
-        :raises: requests.exceptions.HTTPError
-        """
-        return self._query(MWDBFile, hash, raise_not_found)
-
-    def query_config(self, hash, raise_not_found=True):
-        """
-        Queries for configuration object using provided hash
-
-        :param hash: Object hash (SHA-256 identifier)
-        :type hash: str
-        :param raise_not_found: If True (default), method raises HTTPError when object is not found
-        :type raise_not_found: bool
-        :rtype: :class:`MWDBConfig` or None (if raise_not_found=False)
-        :raises: requests.exceptions.HTTPError
-        """
-        return self._query(MWDBConfig, hash, raise_not_found)
-
-    def query_blob(self, hash, raise_not_found=True):
-        """
-        Queries for blob object using provided hash
-
-        :param hash: Object hash (SHA-256 identifier)
-        :type hash: str
-        :param raise_not_found: If True (default), method raises HTTPError when object is not found
-        :type raise_not_found: bool
-        :rtype: :class:`MWDBBlob` or None (if raise_not_found=False)
-        :raises: requests.exceptions.HTTPError
+    def list_remotes(self):
```

Comment:
What do you think about using a more `logic` name? `list_remotes` makes it sounds like the function will print out a listing of the remotes. Maybe something like `get_remotes` or `enumerate_remotes`?